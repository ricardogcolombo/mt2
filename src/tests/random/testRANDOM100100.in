100 100
1 57 14 31 16
1 88 29 74 85
1 41 108 61 55
1 54 99 93 122
1 88 44 44 74
1 19 4 88 6
1 79 48 37 49
1 1 95 62 46
1 10 107 41 45
1 100 11 64 13
1 37 118 35 97
1 7 4 32 51
1 83 76 60 31
1 64 50 38 64
1 57 88 1 102
1 16 37 55 53
1 44 115 72 25
1 21 23 43 94
1 61 84 98 97
1 90 44 30 119
1 20 42 4 25
1 19 68 20 84
1 55 112 90 105
1 80 58 6 112
1 82 18 18 53
1 56 29 33 20
1 34 101 81 114
1 18 130 16 1
1 80 89 54 108
1 69 74 3 120
1 75 82 45 65
1 49 62 39 38
1 77 112 78 25
1 84 6 19 41
1 78 96 35 31
1 6 16 65 8
1 45 114 27 34
1 56 45 85 4
1 74 53 48 74
1 74 11 79 108
1 28 11 84 71
1 23 114 33 89
1 5 102 41 80
1 34 88 83 49
1 26 125 90 19
1 61 17 8 107
1 26 58 34 68
1 42 108 25 11
1 35 47 52 76
1 65 116 52 36
1 8 16 20 18
1 18 62 81 10
1 57 78 32 94
1 27 63 14 57
1 84 25 67 117
1 95 122 52 90
1 38 22 35 114
1 62 100 68 37
1 19 88 26 123
1 66 119 45 109
1 45 29 93 75
1 34 118 46 130
1 37 16 93 74
1 67 102 19 128
1 65 103 47 88
1 59 57 4 90
1 99 124 34 12
1 13 39 27 31
1 43 91 83 77
1 19 68 74 115
1 17 41 94 50
1 96 113 64 31
1 11 44 78 47
1 24 13 66 61
1 30 51 69 8
1 7 122 99 114
1 59 70 86 5
1 72 96 89 69
1 86 34 13 64
1 28 67 6 41
1 93 88 37 43
1 71 95 26 16
1 96 73 85 36
1 84 13 89 86
1 11 60 22 88
1 99 103 51 92
1 57 70 79 24
1 90 96 8 65
1 45 75 69 74
1 76 85 46 38
1 48 83 99 22
1 19 99 33 18
1 59 68 64 9
1 1 60 99 19
1 16 73 13 122
1 64 61 68 130
1 26 22 15 101
1 14 112 60 29
1 54 103 1 13
1 58 24 80 92
