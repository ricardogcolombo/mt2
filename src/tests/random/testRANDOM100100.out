0.682551
0.500000
0.581485
0.572611
0.627524
0.592283
0.548562
0.353883
0.500000
0.627524
0.289083
0.500000
0.711710
0.402708
0.638815
0.327518
0.375000
0.757030
0.392135
0.763726
0.395901
0.596361
0.573357
0.408428
0.375000
0.416444
0.421425
0.470739
0.500000
0.648083
0.636423
0.739458
0.220071
0.397466
0.324115
0.500000
0.584735
0.393872
0.375000
0.500000
0.382572
0.625000
0.687704
0.744206
0.492708
0.517491
0.357028
0.736683
0.625000
0.500000
0.318263
0.406114
0.500000
0.699139
0.668849
0.634865
0.409270
0.306530
0.526718
0.189406
0.157812
0.532105
0.500000
0.251372
0.571085
0.725284
0.420702
0.445869
0.244455
0.500000
0.638815
0.551166
0.500000
0.491944
0.664236
0.672497
0.686657
0.559971
0.397190
0.419590
0.538624
0.419010
0.354915
0.290673
0.319389
0.309607
0.500000
0.425657
0.460460
0.347876
0.500000
0.500000
0.893553
0.625000
0.635371
0.642690
0.500000
0.552604
0.454789
0.250457
